packages/driver-interface/src/services/messaging.ts

```diff
@@ -309,6 +312,7 @@ export const sendPhotoReceived = (telegramId: number): Promise<Message> =>
               text: 'Klar',
               callback_data: JSON.stringify({
                 e: 'delivered',
+                bookingIds,
```

Comment:
callback_data size is limited to 64 bytes, which is not a lot.... This will cause error if more than that

Reply:
Ah, true! Good catch.